tests/quantization/test_functions.py

```diff
@@ -182,19 +182,15 @@ def calc_scale(input_):
                 return scales
 
         @staticmethod
-        def get_range_level(is_signed, is_weights, bits):
```

Comment:
We will need this parameter when we introduce changes for the second part of the task that assumes having a workaround for the saturation issue.